ServiceLambda/src/main/java/com/kenzie/capstone/service/lambda/AddDrink.java

```diff
@@ -50,8 +50,7 @@ public DrinkRequest convert(String body) {
         try {
```

Comment:
For try block, i recommend only putting the stuff that has the potential to output exceptions rather than too much

Reply:
Resolved in Rev 3